docs/camera_conventions.md

```diff
@@ -0,0 +1,50 @@
+# Camera coordinate frames
+
+
+
+## Different coordinate frames for the cameras
+
+The following simulation includes a pair of stereo cameras that are fixed with respect to a camera frame. When commanding the virtual camera arm, the relation between the cameras and the frame will remain fixed, i.e. , transformations `T_FL` and `T_FR` do not change when moving the virtual ECM.  Finding a transformation between the needle local coordinate frame and the left image plane can be accomplish via the following block of code:
```

Comment:
The word simulation here might be confusing with the general AMBF simulation, may be better to use a different word